docs/proposals/configuration.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Configuration Files Design
+
+authors:
+  - "@kadisi"
+  - "@fisherxu"
+  
+approvers:
+  - "@kevin-wangzefeng"
+  - "@sids-b"
+
+creation-date: 2019-10-02
+
+---
+
+* [Configuration Files Design](#configuration-files-design)
+   * [Abstract](#abstract)
+   * [Goals](#goals)
+   * [Proposal](#proposal)
+      * [cloudcore](#cloudcore)
+         * [config struct](#config-struct)
+         * [default config file](#default-config-file)
+      * [edgecore](#edgecore)
+         * [config struct](#config-struct-1)
+         * [default config file](#default-config-file-1)
+
+# Configuration Files Design
+
+
+## Abstract  
+
+Currently, cloudcore/edgecore/edgesite\`s configuration files are in the conf directory at the same level and have 3 configuration files, it is difficult to maintain and extend.
+
+Now KubeEdge use the `beehive` package to analyze configuration files, but it has many drawbacks,when the program is running, it will output a lot of logs first.
```

Comment:
s/Now KubeEdge use the `beehive` package to analyze configuration files/ Currently KubeEdge uses beehive's config package to analyse configuration files.

Reply:
done